src/LogInForm/LogInForm.css

```diff
@@ -41,10 +41,13 @@
   width: 100%;
 }
 
-#supplies-button, #check-in-button, 
-#check-out-button {
-  height: 75px;
```

Comment:
Am I correct in thinking these are updates that have been removed since you last pulled down the master? I recently pushed, and Sarah merged, and I'm not sure if this was an intentional removal, or if it's just a consequence of the diff.